nodestream_akamai/gtm/gtm.py

```diff
@@ -1,87 +1,72 @@
 import logging
-from ipaddress import IPv4Address, ip_address
 
 from nodestream.pipeline.extractors import Extractor
 
+from ..akamai_utils import addresses
 from ..akamai_utils.gtm_client import AkamaiGtmClient
 
+PARSED_PROPERTY_TYPES = [
+    "weighted-round-robin",
+    "weighted-hashed",
+    "weighted-round-robin-load-feedback",
+    "ranked-failover",
+    "failover",
+    "performance",
+]
+DEEPLINK_PREFIX = "https://control.akamai.com/apps/gtm/#/domains/"
+
```

Comment:
same logic as edns, pulled out because we should only ever have one of these.

I **might** be convinced to move them back to being static class properties so they would have to be accessed as `AkamaiGtmExtractor.PARSED_PROPERTY_TYPES` but since they're only used here my first instinct is to just pull them up.